Build-your-first-dApp.md

```diff
@@ -25,22 +25,39 @@ If you would rather learn from a video, we have a recording available of this tu
 
      _The important bits for us are: `1:06 to 4:14`_
 
-   - Click Ethereum Mainnet in the top. Change to the Goerli Tesnet and get a copy of the account's public address on your Metamask Wallet.
+   - Click Ethereum Mainnet at the top. Change to the Goerli Tesnet and get a copy of the account's public address on your Metamask Wallet.
 
 2. **Request some Goerli Tesnet Ether from a faucet loaded into your Metamask Wallet.**
 
    - [Faucet link to request funds](https://faucets.chain.link/)
    - [Blog explaining a faucet and how to use one](https://blog.b9lab.com/when-we-first-built-our-faucet-we-deployed-it-on-the-morden-testnet-70bfbf4e317e)
 
-3. **Install a http server. Use any you like, but we recommend `lite-server` for beginners:**
 
-   - Install Node.js ([Download and Instructions](https://nodejs.org/en/download/))
-   - Install lite-server (with NPM in a terminal/command prompt):
+4. **Install a code editor.**
+
+   - You can use whatever editor you want, but we recommend installing [Visual Studio Code](https://code.visualstudio.com/) since it has some helpful extensions and characteristics. The extensions are **optional**, but we recommend the following:
+
+      - [Solidity](https://marketplace.visualstudio.com/items?itemName=NomicFoundation.hardhat-solidity) - Syntax highlighting for Solidity
+      - [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) - Allows you to run a local server to test your HTML/CSS/JS files
+      - [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) - Code formatter for JS, CSS, and HTML files
+      - [npm IntelliSense](https://marketplace.visualstudio.com/items?itemName=christian-kohler.npm-intellisense) - Autocompletes npm modules in import statements
+      - [IntelliSense for CSS class names in HTML](https://marketplace.visualstudio.com/items?itemName=Zignd.html-css-class-completion) - Autocomplete for CSS classes in HTML files
+      - [gitLeens](https://marketplace.visualstudio.com/items?itemName=naco-siren.githlens) - Shows git blame information in the editor
```

Comment:
I think you meant [GitLens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens) right?

Reply:
Yup, fixed the typo :)